src/pixel/sender.ts

```diff
@@ -4,7 +4,7 @@ import { LiveConnectConfig, EventBus } from '../types'
 import { StateWrapper } from './state'
 import { WrappedCallHandler } from '../handlers/call-handler'
 
-const DEFAULT_AJAX_TIMEOUT = 0
+const DEFAULT_AJAX_TIMEOUT = 5000
```

Comment:
why?
I thought 0 means there is no timeout 
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/timeout

Reply:
We see error logs that suggest that some browser timeout with a timeout of 0.
This is an attempt to make these go away